sycl/include/CL/sycl/property_list.hpp

```diff
@@ -8,14 +8,14 @@
 
 #pragma once
 
+#include <CL/sycl/context.hpp>
```

Comment:
Why do we add this include here?
It's the only change in this file. It's unclear what problem we are solving with this change.